@here/harp-text-canvas/lib/TextCanvas.ts

```diff
@@ -255,7 +255,10 @@ export class TextCanvas {
 
         if (params.material === undefined) {
             this.m_ownsMaterial = true;
-            this.m_material = createSdfTextMaterial({ fontCatalog: params.fontCatalog });
+            this.m_material = createSdfTextMaterial({
+                fontCatalog: params.fontCatalog,
+                glslVersion: this.m_renderer.capabilities.isWebGL2 ? 3.0 : undefined
+            });
```

Comment:
Same here, just pass the renderer capabilities to createSdfTextMaterial and set there directly the THREE.ShaderMaterialParameters' glslVersion. That way you don't need to repeat the same logic below.

Unfortunately, the same code will have to be duplicated in createMaterial and createSdfTextMaterial, cause harp-text-canvas is meant to be a standalone module.